int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -298,40 +305,38 @@ for (i = 0; i < selectSelector.length; i++) {
 }
 for (i = 0; i < buttonSelector.length; i++) {
 	if (buttonSelector[i].name.match(/dwfrm_profile_address_create/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_profile_address_edit/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_singleshipping_shippingAddress_save/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_profile_confirm/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_billing_save/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
-		enableButtonDisable(edqCurrentSubmitButtonSelector, false);
+		edqCurrentSubmitButton = buttonSelector[i];
```

Comment:
In reality `buttonSelector` is a `NodeList`. Rather than go back and forth on variable names too many times now, we'll do it once we refactor everything to TypeScript.

Reply:
Yes! I think we can address this on a refactoring.